modules/post/multi/manage/screenshare.rb

```diff
@@ -0,0 +1,174 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+
+  include Msf::Exploit::Remote::HttpServer
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Multi Manage the screen of the target meterpreter session',
+        'Description' => %q{
+          This module allows you to view and control the screen of the target computer via
+          a local browser window. The module continually screenshots the target screen and
+          also relays all mouse and keyboard events to session.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [ 'timwr'],
+        'Platform' => [ 'linux', 'win', 'osx' ],
+        'SessionTypes' => [ 'meterpreter' ],
+        'DefaultOptions' => { 'SRVHOST' => '127.0.0.1' }
```

Comment:
I just got bitten by using `osx/x64/meterpreter_reverse_http` and `post/multi/manage/screenshare` together as they both bind to the same port by default.

Definitely PEBKAC on my part, but it stopped the module from working and seemed to leave my session in a broken state, and I couldn't kill the session successfully either.

I don't know if we have already have a Github issue to validate that scenario. But maybe a quick fix for now is providing a different default SRVPORT? I don't know what the common practice here is just yet - so let me know your thoughts :+1: